src/components/Dashboard/AddressListSm.vue

```diff
@@ -9,8 +9,40 @@
         <v-expansion-panel-header id="registered-office-panel-toggle">
           <div class="list-item__title">Registered Office</div>
         </v-expansion-panel-header>
+        <!-- A completed filing is required; Don't show address information -->
+        <v-expansion-panel-content class="panel-wrapper pt-0 pb-0" v-if="completedFilingRequired">
+          <v-list class="pt-0 pb-0">
+            <v-list-item class="delivery-address-list-item">
+              <v-list-item-icon class="address-icon mr-0">
+                <v-icon color="primary">mdi-truck</v-icon>
+              </v-list-item-icon>
+              <v-list-item-content>
+                <v-list-item-title class="mb-2 address-title">Delivery Address</v-list-item-title>
+                <v-list-item-subtitle>
+                  <div>
+                    <span class="complete-filing">Complete your filing to display</span>
+                  </div>
```

Comment:
Just curious, was wrapping the spans in divs for spacing?

Reply:
Just a habit of mine to more easily style a group of elements within the div as well as the individual children. In this case it isn't used, but I just structured it this way.

Cleaner to remove?

Reply:
I'm totally fine with it, was just curious!

Reply:
My 2 cents...

If text won't fit on 1 line between a start and end tag then I'll wrap it in a span, so that, during testing, the exact text can be selected... otherwise you sometimes get extra whitespace in the result.